test/Doctest.hs

```diff
@@ -2,7 +2,12 @@
 
 module Main (main) where
 
-#if MIN_VERSION_base(4,10,1)
+#if defined(mingw32_HOST_OS) || __GLASGOW_HASKELL__ < 802
```

Comment:
So doctests are not working on windows?

Reply:
@vrom911 Yes, you're right. I think it's not so important to run `doctest` on Windows. I think it's enough to run `doctest` only for latest version of GHC and only on single arch. I would love to solve this problem inside `.cabal` file but I can't do this. I've opened issue for this:

* https://github.com/haskell/cabal/issues/5055